### PR TITLE
update(docs): Week 1,2 and 3 updated

### PR DESCRIPTION
**Pull request by soham4abc**
## Project

Name of the project : REST API completion

### Changes

Weekly update for week 1, 2 and 3

### Notes

@GMishx @shaheemazmalmmd

### CODE REVIEW

File: docs/2023/rest/updates/soham/2023-06-01.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+SPDX-License-Identifier: CC-BY-SA-4.0
+
+SPDX-FileCopyrightText: 2023 Soham Banerjee <sohambanerjee4abc@hotmail.com>
+-->
+
+## Activities
+
+- Issued Pull Request for the OpenAPI endpoint. `/openapi`: [#2474](https://github.com/fossology/fossology/pull/2474).
+
+### SOME SCREENSHOTS
```
Comment (avinal) [inline docs/2023/rest/updates/soham/2023-06-01.md:19]:
```suggestion
### SOME SCREENSHOTS
```
Is this some format decided by you all? 😅 cc @dushimsam, apologies if I am not aware

Comment (soham4abc) [reply docs/2023/rest/updates/soham/2023-06-01.md:19]: No actually.... I tried to follow his pattern thinking that it was the standard as it was merged... :-)

Comment (soham4abc) [reply docs/2023/rest/updates/soham/2023-06-01.md:19]: Should I change it?? @avinal

Comment (avinal) [reply docs/2023/rest/updates/soham/2023-06-01.md:19]: No, it is fine, it doesn't make much difference as long as it is correct information.

Comment (soham4abc) [reply docs/2023/rest/updates/soham/2023-06-01.md:19]: Thank you @avinal !!

**Comment by soham4abc** [comment]
@avinal I have updated the docs as per suggestions. Does this look good?

**Review by avinal — APPROVED** [review]
Looks good to me

**Review by GMishx — APPROVED** [review]

**Pull request merged**